docs/usage/configuration-options.md

```diff
@@ -2446,6 +2446,17 @@ In those cases a feature request needs to be implemented.
 !!! warning "Warning for Maven users"
     For `minimumReleaseAge` to work, the Maven source must return reliable `last-modified` headers.
 
+    <!-- markdownlint-disable MD046 -->
```

Comment:
I disabled the rule, because it seems to be false negative. because it treats continuation of admonition as a indented code which it is not intended to be, but at the same time MkDocs documentation on admonitions is not clear if multi paragraph admonitions are even possible https://squidfunk.github.io/mkdocs-material/reference/admonitions/#inline-blocks-inline-end

```
MD046/code-block-style Code block style [Expected: fenced; Actual: indented] https://github.com/DavidAnson/markdownlint/blob/v0.37.4/doc/md046.md
```